Remotion/Data/DomainObjects.UnitTests/DatabaseTest.cs

```diff
@@ -69,11 +70,14 @@ public virtual void TearDown ()
     [OneTimeSetUp]
     public virtual void OneTimeSetUp ()
     {
+      TestDomainDatabase.InitializeDatabase();
+      StandardConfiguration.Instance.EnableConnections();
```

Comment:
Calling `StandardConfiguration.Instance` here without having `StandardConfiguration.Initialize()` is "wrong". `StandardConfiguration.Instance` would fail without `StandardConfiguration.Initialize()`.

Reply:
In this case the initialization of the `StandardConfiguration` is done in `TestDomainDatabase.InitializeDatabase()`

Reply:
Then we should just do it twice (it's idempotent anyway) and thus make it obvious that the intiialization happens regardless of TestDomainDatabase.

I think this might also be a good example why an Uninitialize could be helpful. The DB-Setup needs the standardconfig, but after it's done, removing Instance until it's done explicitly in a test setup would remove potential sideeffects.

Reply:
The initialization is not quite idempotent as it recreates a new instance of `StandardConfiguration`. But apart from that I don't think that there are any effects on the tests. As mentioned in the comment below I would prefer to do this in another issue.

Reply:
Okay, so, separate ticket then (please create). btw: the idea is to keep the reference to this instance alive internally, so there is no new instance between swithcing it off and onagain..

Reply:
Arrgg. Implicit base constructor calls paired with doing work in the constructor is painful. Tests took 1min longer when I put the `StandardConfiguration.Initialize()` into the `OneTimeSetUp`. Turns out, the base constructor is not quite indempotent and recreates the `MappingConfiguration` from the reflection....

I changed the `Initialize` methods to `EnsureInitialized` to prevent duplicate creation and I also found that the same `DisableConnection` stuff should be done on the `TableInheritanceConfiguration`.